MANIFEST.in

```diff
@@ -0,0 +1 @@
+include *.json
```

Comment:
What is this file and content for?

Reply:
There's a `collection-asset-metadata.json` file that I want to include in the package, so we don't have to read the title and descriptions from the NetCDFs every time we make a collection.

Reply:
Yeah. Falling into a hole as to why to use this rather than `options.package_data` in setup.cfg. Seems related to one being valid for source distributions (MANIFEST.in) and the other (options.package_data) for binary, but plenty of conflicting info out there.

Reply:
Hm, it's a good point -- I wonder if I duplicated effort by specifying `package_data` _and_ `Manifest.in`. To be honest I have to re-lookup how to do this stuff every time.

Reply:
Erf. I didn't even look to see that you included it in setup.cfg as well.